src/components/bcros/todo/Item.vue

```diff
@@ -48,7 +49,7 @@ const showDetailsBtnRed = (item: TodoItemI): boolean => {
             variant="ghost"
             leading-icon="i-mdi-information-outline"
             class="-mt-1 h-8"
-            :class="showDetailsBtnRed(item) ? 'text-red-500' : ''"
+            :class="useErrorStyle(item) ? 'text-red-500 hover:bg-red-100' : ''"
```

Comment:
this was already in place, but fyi red-600 is the bc registries wide error colour